README_Docker.md

```diff
@@ -9,6 +9,10 @@ $apt-cache policy docker-engine
 $sudo apt-get install -y docker-engine
 Docker should now be installed and ready to use
 
+For proxy users, please also make the following change
+https://docs.docker.com/engine/admin/systemd/#http-proxy (HTTP proxy section)
+Create a common network for the containers
+
 Create a common network for the containers
 ------------------------------------------
 $docker network create node-dc-net
```

Comment:
Sure, will take care of the formatting.

The dockerfiles themselves are in this commit
https://github.com/Node-DC/Node-DC-EIS/pull/3/commits/04abb9be9604d6e6d23b01ad34db0f636b149e49

I thought this would show up in the latest commit, for some reason it hasn't.